code/about.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <menu>
+        <a href=./index.html>Home</a>
+        <a href=./help.html>Das kannst du tun</a>
+        <a href=./newhome.html>Zuhause gesucht</a>
+        <a href=./about.html>Über uns</a>
+    </menu>
+    <link rel="stylesheet" type="text/css" href="style.css"/>
+</head>
+<header>
+    <h1>FILOSS</h1>
+</header>
+<body>
+        <h2>Der Verein</h2>
+            <h3>Unser Anliegen</h3>
+                <p>Griechenland hat ein strenges Tierschutzgesetz. Wer dieses ignoriert oder nicht einhält, muss mit drakonischen Geldbussen rechnen. Und dennoch werden die Rechte der Tiere dauernd und in schwerster Weise verletzt.
+                    Zum Zweck der Mittelbeschaffung besteht deshalb seit Juli 2016 der gemeinnützige Verein zur Förderung des Tierschutzverein Serifos FILOSS. Systematisch und organisiert setzen wir uns vor Ort für die Tierrechte auf Serifos ein und möchten den Aufbau eines Tierheims und einer Tierpraxis umsetzen.
+                </p>    
+            <h3>Wir setzen uns zum Ziel:</h3>
+                    <ul>Kätzchen und Welpen aufzunehmen</ul>
+                    <ul>kranke und gebrechliche Nutztiere (Ziegen, Schafe, Esel, Mulis und Pferde) unterzubringen und zu pflegen</ul>
+                    <ul>eine tiermedizinische Praxis aufzubauen, um Jungtiere zu impfen</ul>
+                    <ul>geschlechtsreife Hunde und Katzen zu sterilisieren</ul>
+                    <ul>medizinische Hilfe zu leisten</ul>
+                    <ul>die Bevölkerung allgemein für die Anliegen des Tierschutzes zu sensibilisieren</ul>
+           
+                    
+                <p>((Statuten download))</p>
+            
+    
+        <h2>Hunde-Hof Maima Serifos</h2>
+            <h3>Ein Heim für Hunde</h3>
```

Comment:
Remember that all the elements that are siblings should be at the same level of indentation. 🧑‍🤝‍🧑